platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/snapshotter/MapSnapshotter.java

```diff
@@ -29,22 +29,24 @@
 import timber.log.Timber;
 
 /**
- * The map snapshotter creates a large of the map, rendered
- * off the UI thread. The snapshotter itself must be used on
- * the UI thread (for access to the main looper)
+ * Use MapSnapshotter to create static map images, rendered in the background.
+ * <br>
```

Comment:
the usage of <br> has led to javadoc generation issues in the past, we tend to use `<p> ... </p>` instead